supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -407,6 +418,17 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*:
 
+[discrete]
+[[appstream-repository]]
+==== image:images/yes.png[yes] AppStream repository (noun)
+*Description*: RHEL content in the _AppStream repository_ includes Application Streams, such as additional user-space applications, runtime languages, and databases in support of varied workloads and use cases.
```

Comment:
The current version defines _RHEL content_ rather than _AppStream repository_. How about:
```suggestion
*Description*: The _AppStream repository_  distributes RHEL content as Application Streams and other components. These Application Streams include additional user-space applications, runtime languages, and databases for various workloads and use cases.
```